src/Neo/SmartContract/Native/ContractManagement.cs

```diff
@@ -87,10 +47,13 @@ private int GetNextAvailableId(DataCache snapshot)
             return value;
         }
 
-        internal override ContractTask Initialize(ApplicationEngine engine)
+        internal override ContractTask Initialize(ApplicationEngine engine, Hardfork? hardfork)
         {
-            engine.Snapshot.Add(CreateStorageKey(Prefix_MinimumDeploymentFee), new StorageItem(10_00000000));
-            engine.Snapshot.Add(CreateStorageKey(Prefix_NextAvailableId), new StorageItem(1));
+            if (hardfork is null)
```

Comment:
maybe add a comment:

"If there is any hardfork it means that ContractManagement was already initialized, then, skip this.¨